shared/chat/blocking/block-modal/container.tsx

```diff
@@ -0,0 +1,98 @@
+import * as Container from '../../../util/container'
+import * as RouteTreeGen from '../../../actions/route-tree-gen'
+import BlockModal, {BlockType, NewBlocksMap, ReportSettings} from '.'
+import * as UsersGen from '../../../actions/users-gen'
+import * as TeamsGen from '../../../actions/teams-gen'
+import * as Constants from '../../../constants/users'
+import {leaveTeamWaitingKey} from '../../../constants/teams'
+
+type OwnProps = Container.RouteProps<{
+  blockByDefault?: boolean
+  convID?: string
+  others?: Array<string>
```

Comment:
What is `others`?

Reply:
thanks, this broke and I didn't notice

Reply:
for reference, it's to drive this:
![image](https://user-images.githubusercontent.com/451974/69584114-ae513200-0fdc-11ea-9f64-7f62a2918f68.png)
if you click "Block" from impteam chat with more than two people